mod/cp_notifications/start.php

```diff
@@ -1212,18 +1285,21 @@ function cp_create_notification($event, $type, $object) {
 
 			$to_recipients = get_subscribers($dbprefix, $author_id, $content_id);
 			$to_recipients_site = get_site_subscribers($dbprefix, $author_id, $content_id);
-			$email_only = false;
 			break;
 
 	} // end of switch statement
 
 
+$to_recipients_site = null;
```

Comment:
Indent

Reply:
i forgot to remove (its there to test my conditions)